src/main/java/duke/Command.java

```diff
@@ -0,0 +1,85 @@
+package duke;
+
+import duke.exception.InvalidCommandException;
+import duke.task.Task;
+import duke.task.TaskManager;
+
+class Command {
+
+    private enum Commands {
+        BYE("bye", 1),
```

Comment:
string literals like `"bye"`, `"list"` should be defined as named constants

Reply:
Hi Tejas, just to clarify. I've tied the use of the string literal to the enum which is a constant? If I made a named constant only to have them reference them in the enum would that not be redundant?